cmd/main.go

```diff
@@ -352,7 +354,7 @@ func main() {
 
 	bm := bminventory.NewBareMetalInventory(db, log.WithField("pkg", "Inventory"), hostApi, clusterApi, Options.BMConfig,
```

Comment:
It becomes too crowded. Can we consider the Builder pattern for instantiating BM inventory?

Reply:
i think we should discuss it, but not in this PR :)